db/migrate/20221014130631_remove_content_from_lessons.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveContentFromLessons < ActiveRecord::Migration[6.1]
```

Comment:
Perhaps better to do the removal in a follow up PR? May have to watch and manually run a content update in order to re-seed all the content, too

Reply:
Totally to include this @ChargrilledChook, the seeds and import tasks are run as part of the release phase on Heroku. That phase needs to complete successfully before the release goes live so it should be safe.